### DISCUSSION TITLE
zh/docs/rbac-with-pattern

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1167] [Docs comments]
# zh/docs/rbac-with-pattern

RBAC with Pattern

https://casbin.org/zh/docs/rbac-with-pattern

**Comment by selfuppen** [comment]
If there are some other languages'(like Java) demo it will be very helpful.

**Reply by selfuppen** [reply]
Eureka, Java examples:
https://github.com/casbin/jcasbin/blob/master/src/test/java/org/casbin/jcasbin/main/RbacAPIWithPatternMatchUnitTest.java
